modules/private-clusters-setting-dns-private.adoc

```diff
@@ -42,22 +49,24 @@ status: {}
 +
 Note that the `spec` section contains both a private and a public zone.
 
-. Patch the `DNS` custom resource to remove the public zone:
+. Patch the `DNS` CR to remove the public zone:
 +
 [source,terminal]
 ----
 $ oc patch dnses.config.openshift.io/cluster --type=merge --patch='{"spec": {"publicZone": null}}'
 dns.config.openshift.io/cluster patched
 ----
 +
-Because the Ingress Controller consults the `DNS` definition when it creates `Ingress` objects, when you create or modify `Ingress` objects, only private records are created.
+The Ingress Operator consults the `DNS` CR definition when it creates DNS records for `IngressController` objects. If only private zones are specified, only private records are created.
 +
 [IMPORTANT]
 ====
-DNS records for the existing Ingress objects are not modified when you remove the public zone.
+Existing DNS records are not modified when you remove the public zone. You must manually delete previously published public DNS records if you no longer want them to be published publicly.
```

Comment:
Not even sure what to think of as an alternative, but just want to note that "previously published public" is a bit of a tongue twister haha 😅